[2024-05-07T07:27:52.119+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-07T07:27:52.151+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: snowflake_connector3.count_query manual__2024-05-07T07:27:40.596943+00:00 [queued]>
[2024-05-07T07:27:52.164+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: snowflake_connector3.count_query manual__2024-05-07T07:27:40.596943+00:00 [queued]>
[2024-05-07T07:27:52.165+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-05-07T07:27:52.183+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): count_query> on 2024-05-07 07:27:40.596943+00:00
[2024-05-07T07:27:52.191+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=254) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-07T07:27:52.193+0000] {standard_task_runner.py:63} INFO - Started process 256 to run task
[2024-05-07T07:27:52.193+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'snowflake_connector3', 'count_query', 'manual__2024-05-07T07:27:40.596943+00:00', '--job-id', '177', '--raw', '--subdir', 'DAGS_FOLDER/latihan.py', '--cfg-path', '/tmp/tmp3zphobgf']
[2024-05-07T07:27:52.196+0000] {standard_task_runner.py:91} INFO - Job 177: Subtask count_query
[2024-05-07T07:27:52.261+0000] {task_command.py:426} INFO - Running <TaskInstance: snowflake_connector3.count_query manual__2024-05-07T07:27:40.596943+00:00 [running]> on host 3078ffe3180a
[2024-05-07T07:27:52.374+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Airflow' AIRFLOW_CTX_DAG_ID='snowflake_connector3' AIRFLOW_CTX_TASK_ID='count_query' AIRFLOW_CTX_EXECUTION_DATE='2024-05-07T07:27:40.596943+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-07T07:27:40.596943+00:00'
[2024-05-07T07:27:52.376+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-07T07:27:52.392+0000] {base.py:84} INFO - Using connection ID 'snowflake_conn' for task execution.
[2024-05-07T07:27:52.393+0000] {connection.py:386} INFO - Snowflake Connector for Python Version: 3.7.1, Python Version: 3.12.2, Platform: Linux-5.15.146.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-05-07T07:27:52.394+0000] {connection.py:1211} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-05-07T07:27:52.395+0000] {connection.py:1221} INFO - THIS CONNECTION IS IN INSECURE MODE. IT MEANS THE CERTIFICATE WILL BE VALIDATED BUT THE CERTIFICATE REVOCATION STATUS WILL NOT BE CHECKED.
[2024-05-07T07:27:52.568+0000] {ssl_wrap_socket.py:100} INFO - THIS CONNECTION IS IN INSECURE MODE. IT MEANS THE CERTIFICATE WILL BE VALIDATED BUT THE CERTIFICATE REVOCATION STATUS WILL NOT BE CHECKED.
[2024-05-07T07:27:53.079+0000] {cursor.py:1032} INFO - query: [ALTER SESSION SET autocommit=False]
[2024-05-07T07:27:53.410+0000] {cursor.py:1045} INFO - query execution done
[2024-05-07T07:27:53.411+0000] {cursor.py:1205} INFO - Number of results in first chunk: 1
[2024-05-07T07:27:53.412+0000] {sql.py:457} INFO - Running statement: select count(*) from dbt_snowflake.public_raw.customer, parameters: None
[2024-05-07T07:27:53.412+0000] {cursor.py:1032} INFO - query: [select count(*) from dbt_snowflake.public_raw.customer]
[2024-05-07T07:27:54.207+0000] {cursor.py:1045} INFO - query execution done
[2024-05-07T07:27:54.208+0000] {cursor.py:1205} INFO - Number of results in first chunk: 1
[2024-05-07T07:27:54.208+0000] {sql.py:466} INFO - Rows affected: 1
[2024-05-07T07:27:54.210+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/providers/snowflake/hooks/snowflake.py:393: AirflowProviderDeprecationWarning: Call to deprecated method _make_common_data_structure. (The `_make_serializable` method is deprecated and support will be removed in a future version of the common.sql provider. Please update the DbApiHook's provider to a version based on common.sql >= 1.9.1.)
  result = self._make_common_data_structure(handler(cur))  # type: ignore[attr-defined]

[2024-05-07T07:27:54.211+0000] {snowflake.py:402} INFO - Rows affected: 1
[2024-05-07T07:27:54.212+0000] {snowflake.py:403} INFO - Snowflake query id: 01b429ff-0002-1353-0002-b462000de032
[2024-05-07T07:27:54.213+0000] {cursor.py:1032} INFO - query: [COMMIT]
[2024-05-07T07:27:54.548+0000] {cursor.py:1045} INFO - query execution done
[2024-05-07T07:27:54.549+0000] {cursor.py:1205} INFO - Number of results in first chunk: 1
[2024-05-07T07:27:54.549+0000] {connection.py:734} INFO - closed
[2024-05-07T07:27:54.896+0000] {connection.py:740} INFO - No async queries seem to be running, deleting session
[2024-05-07T07:27:55.219+0000] {latihan.py:27} INFO - Number of rows in `dbt_dev.public_raw.customer`  - 31
[2024-05-07T07:27:55.223+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-07T07:27:55.224+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-07T07:27:55.267+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=snowflake_connector3, task_id=count_query, execution_date=20240507T072740, start_date=20240507T072752, end_date=20240507T072755
[2024-05-07T07:27:55.343+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-07T07:27:55.371+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-07T07:27:55.375+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
